packages/gatsby/src/joi-schemas/joi.js

```diff
@@ -59,6 +59,7 @@ export const nodeSchema = Joi.object()
       fieldOwners: Joi.array(),
       content: Joi.string().allow(``),
       description: Joi.string(),
+      rawMarkdownBody: Joi.string(),
```

Comment:
We don't want to change this schema for single plugin - as mentioned in previous comment, move new property out of `internal` and remove this schema change